### PR TITLE
add optimism to beets and fix bugs

### PR DESCRIPTION
**Pull request by sevenshi**

### CODE REVIEW

File: subgraphs/balancer-v2/config/paramConfig.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { dataSource, BigDecimal, log } from "@graphprotocol/graph-ts";
+
+
+let UNTRACKEDPOOLSTEMP:string[]
```
Comment (this-username-is-taken) [inline subgraphs/balancer-v2/config/paramConfig.ts:4]: formatting

File: subgraphs/balancer-v2/config/paramConfig.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+if (dataSource.network() == "mainnet") {
+  UNTRACKEDPOOLSTEMP=["0x5a172919fff040196b36b238b9e2e7e6d33f616a"
```
Comment (this-username-is-taken) [inline subgraphs/balancer-v2/config/paramConfig.ts:7]: formatting

File: subgraphs/balancer-v2/config/paramConfig.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+                    ,"0xC065798F227b49C150bCDC6CDc43149A12c4d757"]
+}
+
+ 
+
+export namespace NetworkConfigs {
+  export const UNTRACKEDPOOLS = UNTRACKEDPOOLSTEMP;
```
Comment (this-username-is-taken) [inline subgraphs/balancer-v2/config/paramConfig.ts:14]: should be `UNTRACKED_POOLS` and `UNTRACKED_POOLS_TEMP`

**Comment by this-username-is-taken** [comment]
Good update 👍

You should use `prettier` to make sure everything's formatted correctly

File: subgraphs/balancer-v2/src/common/constants.ts [hunk 4]
```diff
@@ -130,20 +128,22 @@ export const PROTOCOL_FEE_TO_OFF = BigDecimal.fromString("0");
 ///// Protocol Specific /////
 /////////////////////////////
 
-export let DEFAULTNETWORK = "FANTOM";
+export let DEFAULTNETWORK = "OPTIMISM";
```
Comment (this-username-is-taken) [inline subgraphs/balancer-v2/src/common/constants.ts:131]: change to `DEFAULT_NETWORK`

File: subgraphs/balancer-v2/src/common/constants.ts [hunk 5]
```diff
@@ -130,20 +128,22 @@ export const PROTOCOL_FEE_TO_OFF = BigDecimal.fromString("0");
...
 
-export let VAULT_ADDRESS = Address.fromString("0x20dd72Ed959b6147912C2e529F0a0C651c33c9ce");
+export let VAULT_ADDRESS = Address.fromString("0xBA12222222228d8Ba445958a75a0704d566BF2C8");
 
-export let MASTERCHEFV2_ADDRESS = Address.fromString("0x8166994d9ebBe5829EC86Bd81258149B87faCfd3");
+export let REWARDTOKEN = "";
```
Comment (this-username-is-taken) [inline subgraphs/balancer-v2/src/common/constants.ts:135]: change to `REWARDTOKEN`

File: subgraphs/balancer-v2/src/mappings/masterchef.ts [hunk 6]
```diff
@@ -74,9 +81,11 @@ export function handleLogUpdatePool(event: LogUpdatePool): void {
     return;
   }
   let pool = LiquidityPool.load(poolAddress.value.toHexString());
-  if(!pool){
-    if(FBEETS.toHexString() == poolAddress.value.toHexString()){
-      pool = createLiquidityPool(event,poolAddress.value.toHexString(),"FreshBeets","FBEETS",[FBEETS.toHexString()] )
+  if (!pool) {
+    if (FBEETS.toHexString() == poolAddress.value.toHexString()) {
```
Comment (this-username-is-taken) [inline subgraphs/balancer-v2/src/mappings/masterchef.ts:85]: what's `FBEETS`?

Comment (sevenshi) [reply subgraphs/balancer-v2/src/mappings/masterchef.ts:85]: [fBEETS)](https://ftmscan.com/token/0xfcef8a994209d6916eb2c86cdd2afd60aa6f54b1)
FBEETS is a token that can be stacked 
but not create by the vault

Comment (this-username-is-taken) [reply subgraphs/balancer-v2/src/mappings/masterchef.ts:85]: I see

**Comment by this-username-is-taken** [comment]
Thanks for the changes!

**Pull request merged**